bcal.c

```diff
@@ -621,6 +621,68 @@ bool lba2chs(char *lba, t_chs *p_chs)
 	return TRUE;
 }
 
+/* Convert any unit in bytes */
+maxuint_t unitconv(char *num,char *unit)
```

Comment:
Please add a space between the two args. Indentation is very important for other people to understand your code.